### PR TITLE
Update scaffold.dart

### PR DESCRIPTION
**Pull request by farmery**
in the original file the option to change the frontlayer's color wasn't available from the scaffold's constructor so i made it possible here.

### CODE REVIEW

**Comment by daadu** [comment]
@WieFel Do you think this is needed? Can't applying background color to `backLayer` widget do the same?

**Comment by WieFel** [comment]
> @WieFel Do you think this is needed? Can't applying background color to `backLayer` widget do the same?

LGTM. We already have `backLayerBackgroundColor` and the suggestion by @farmery would be to add the same for the front layer. `frontLayerBackgroundColor` and `backLayerBackgroundColor` could both be avoided if the user would wrap the back-/front-layer in a `Column` or similar and colors the background of it.

However, I think it's more convenient to have the two color properties present.

File: lib/scaffold.dart [hunk 1]
```diff
@@ -140,6 +140,8 @@ class BackdropScaffold extends StatefulWidget {
   ///
   /// Defaults to `Theme.of(context).primaryColor`.
   final Color backLayerBackgroundColor;
+  
+  final Color frontLayerBackgroundColor;
```
Comment (WieFel) [inline lib/scaffold.dart:144]: @farmery please add documentation similar to `backLayerBackgroundColor`!

File: lib/scaffold.dart [hunk 2]
```diff
@@ -499,6 +502,7 @@ class BackdropScaffoldState extends State<BackdropScaffold>
 
   Widget _buildFrontPanel(BuildContext context) {
     return Material(
+      color: this.widget.frontLayerBackgroundColor?? Colors.white,
```
Comment (WieFel) [inline lib/scaffold.dart:505]: @farmery please remove the `?? Colors.white`.

Docs of `Material.color` say: "By default, the color is derived from the [type] of material."
So, if `widget.frontLayerBackgroundColor` is not set (= is null), `Material` handles the initialisation of the color itself.

**Comment by daadu** [comment]
@farmery Please do necessary changes. Awaiting for it.

**Comment by daadu** [comment]
@WieFel Apply the changes yourself, and let's merge this, so that it lands in the next release.

**Comment by daadu** [comment]
@WieFel  You will have to push changes to `farmery:patch-1` or use Github UI to make changes.

**Comment by daadu** [comment]
This feature has landed with [`v0.4.7` on pub.dev](https://pub.dev/packages/backdrop/versions/0.4.7).

@farmery Check yourself on README!!

**Comment by farmery** [comment]
thank you very much! so sorry for not responding all these while.

**Pull request merged**